lib/actions/request_review.js

```diff
@@ -0,0 +1,64 @@
+const { Action } = require('./action')
+const _ = require('lodash')
+
+const createRequestReview = async (context, number, reviewers) => {
+  return context.github.pulls.createReviewRequest(
+    context.repo({ pull_number: number, reviewers })
+  )
+}
+
+const fetchCollaborators = async (context) => {
+  return context.github.repos.listCollaborators(
+    context.repo()
+  )
+}
+
+class RequestReview extends Action {
+  constructor () {
+    super()
+    this.supportedEvents = [
+      'pull_request.opened',
+      'pull_request.edited',
+      'pull_request_review.submitted',
+      'pull_request_review.edited',
+      'pull_request_review.dismissed',
+      'pull_request.labeled',
+      'pull_request.milestoned',
+      'pull_request.demilestoned',
+      'pull_request.assigned',
+      'pull_request.unassigned',
+      'pull_request.unlabeled',
+      'pull_request.synchronize'
```

Comment:
Would it make sense to specify `pull_request.*` instead?

Reply:
done